### ISSUE TITLE
panorama.dispose() on video gallery

### ISSUE THREAD

**Issue by baseline-s** [pchen66/panolens.js#54]
Is it possible to switch panorama's by calling:

panorama.dispose();
viewer.remove( panorama );

panorama = new PANOLENS.VideoPanorama( this.url, { autoplay: true, loop:true, muted:false } );
viewer.add( panorama );
viewer.setPanorama( panorama );

I am using your gallery example as a base, but cannot seem to delete the video object and switch to a new panorama. The current one continues to run in the background. Anyone experienced this ?

**Comment by smitshah81** [comment]
To delete video object before disposing panora you can do something like below.
```
if(typeof viewer.panorama.VideoElement != "undefined")
{
viewer.panorama.VideoElement.src="";
}
```
I hope this works for you :)

**Comment by baseline-s** [comment]
Thanks, but this has the same effect and the video continues to play in the background.

**Comment by smitshah81** [comment]
I guess inthat case you have to add

**Comment by pchen66** [comment]
`VideoPanorama` now has proper `dispose` method to cleanup state and video playing. Closed #82
